.github/workflows/pr-labeler.yaml

```
@@ -0,0 +1,11 @@
+name: pr-labeler
+on: [pull_request]
```

To be consistent with other workflow files.
```suggestion
on
  - pull_request:
```

---

Ok, reflected it by https://github.com/autowarefoundation/autoware.universe/pull/2266/commits/2c00377ee913fd7c3d13fdd40e892493116492db